src/parser/paladin/protection/CombatLogParser.ts

```diff
@@ -34,6 +35,7 @@ class CombatLogParser extends CoreCombatLogParser {
     // Spells
     lightOfTheProtector: LightOfTheProtector,
     hotr: HammerOfTheRighteous,
+    wog: WordOfGlory,
```

Comment:
I think it's nicer to write it out, much easier to debug by anyone including people less familiar with the spec should you run into issues.